mcr.cpp

```diff
@@ -23,36 +23,46 @@ return win;
 } 
 int main(){
 int i, j;
-char game[3][3] = {' '}; // Tic-tac-toe
+	char game[3][3] = {
+        {' ', ' ', ' '},
+        {' ', ' ', ' '},
+        {' ', ' ', ' '}
+    };
 char player1 = 'X'; 
 char player2 = 'O'; 
-bool turn = true; // false for player 1's turn, true for player 2's turn. 
-Player 1 first.
+bool turn = true; // false for player 1's turn, true for player 2's turn. Player 1 first.
 cout << "X = Player 1" << endl << "O = Player 2" << endl;
  for (int n=0; n<9; n++){
  turn = !turn; // use the not-operator to change true to false or 
 false to true.
  if (turn == false) 
  cout << "Player 1: "; 
  else
- cout << "Player 2: "; 
+  cout << "Player 2: "; 
  cout << "Which cell to mark? i:[1..3], j:[1..3]: "; 
  cin >> i >> j;
- if (turn == false) 
- game[i][j] = 'X';
-else 
- game[i][j] = 'O'; 
- if (isWin(game)){
- cout << "Win!" << endl;
- break; // need to terminate the problem
- } 
- } 
-if (i==3) // all celles with i=0 to 2 have been inputted above but no 
-winner yet
- cout << "Tie!" << endl;
+while( i>2 || j>2 || i<0 || j<0 ) 
+		   {
+			cout<<"Wrong input,please mark it again "<<endl;
+			cin>>i>>j;
```

Comment:
### Add a **break;** into circulation
To avoid incorrect input from causing endless loops in the program
`while( i>2 || j>2 || i<0 || j<0 ) 
		   {
			cout<<"Wrong input,please mark it again "<<endl;
                        break;
			cin>>i>>j;}`